cfme/tests/infrastructure/test_host.py

```diff
@@ -21,13 +21,15 @@
 from cfme.markers.env_markers.provider import ONE_PER_TYPE
 from cfme.markers.env_markers.provider import ONE_PER_VERSION
 from cfme.tests.networks.test_sdn_downloads import handle_extra_tabs
+from cfme.utils.appliance import IPAppliance
```

Comment:
_Optional_ since you only use `DownloadOptions` from `IPAppliance` why not only import that?

Reply:
I did suggest putting this _in_ IPAppliance, but that does create an odd import when you want to use it for test parametrization.

I'm thinking its worth creating a `cfme.utils.appliance.constants` module, for defining these constants/enums in their own importable module.

Reply:
Moved as suggested